docs/guides/containers/podman_guide.md

```diff
@@ -10,7 +10,7 @@ tags:
 
 # Introduction
 
-!!! note
+!!! note "Note"
```

Comment:
Hi @ChrSteinert, I'm commenting inline here. There's no need to add the "Note" in quotes as you have done. It is an option for translators to add that in their language if they want within the Crowdin interface, however most have chosen *not* to do this, as in general, the admonition titles, even in English, are pretty self-explanatory. I'd prefer that you not add these within the English source documents.